crates/vm2/src/program.rs

```diff
@@ -75,3 +116,36 @@ impl<T, W> PartialEq for Program<T, W> {
             && Arc::ptr_eq(&self.instructions, &other.instructions)
     }
 }
+
+/// "Jump to start" instruction placed at the end of programs exceeding `1 << 16` instructions.
```

Comment:
```suggestion
/// Wraparound instruction placed at the end of programs exceeding `1 << 16` instructions to simulate the 16-bit program counter overflowing.
/// Does not invoke tracers because it is an implementation detail, not an actual instruction.
```